packages/client-sdk-nodejs/src/config/middleware/cache-request-logging.ts

```diff
@@ -366,7 +366,7 @@ const convertSetUnionRequest: RequestToLogInterfaceConverterFn<
 
 interface SetDifferenceRequestLog extends SetCollectionRequestLog {
   action: 'minuend' | 'subtrahend_set' | 'subtrahend_identity';
-  elements?: string[];
+  elements: string[] | null;
```

Comment:
Similarly, this is a breaking change if the type is exposed, and we should prefer `?` notation.

Reply:
same response for all of the stuff in this file - the stuff in this level is very low-level middleware stuff that I'm not expecting any users to touch directly.